src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/CreateAccessRequirementButton.java

```diff
@@ -33,49 +38,25 @@ public class CreateAccessRequirementButton implements IsWidget {
 
   @Inject
   public CreateAccessRequirementButton(
-    Button button,
+    SingleButtonView view,
     IsACTMemberAsyncHandler isACTMemberAsyncHandler,
+    CookieProvider cookies,
     final PortalGinInjector ginInjector
   ) {
-    this.button = button;
+    this.view = view;
     this.isACTMemberAsyncHandler = isACTMemberAsyncHandler;
     this.ginInjector = ginInjector;
-    button.setVisible(false);
-    button.addStyleName("margin-left-10");
-    button.addClickHandler(
-      new ClickHandler() {
-        @Override
-        public void onClick(ClickEvent event) {
-          CreateAccessRequirementWizard wizard =
-            ginInjector.getCreateAccessRequirementWizard();
-          if (subject != null) {
-            wizard.configure(subject);
-          } else if (ar != null) {
-            wizard.configure(ar);
-          }
-          wizard.showModal(
-            new WizardCallback() {
-              @Override
-              public void onFinished() {
-                refreshCallback.invoke();
-              }
-
-              @Override
-              public void onCanceled() {
-                refreshCallback.invoke();
-              }
-            }
-          );
-        }
-      }
-    );
```

Comment:
Moved into `useSwcWizard`, so we can conditionally show the SWC or SRC wizard based on whether the user is in Experimental mode